libopenage/renderer/camera/camera.h

```diff
@@ -84,7 +85,9 @@ class Camera {
 	 *
 	 * @param scene_pos New 3D position of the camera in the scene.
 	 */
-	void move_to(Eigen::Vector3f scene_pos);
+	void move_to(Eigen::Vector3f scene_pos, std::optional<std::pair<float, float>> x_bounds = std::nullopt,
+											std::optional<std::pair<float, float>> z_bounds = std::nullopt);
```

Comment:
I'm not sure if you code formatted this but if you didn't try running `clang-format` over this file. The arguments should either be placed on the same line or in this case on a separate line each.